packages/lesswrong/components/sunshineDashboard/SunshineNewUsersInfo.tsx

```diff
@@ -360,6 +368,14 @@ const SunshineNewUsersInfo = ({ user, classes, updateUser }: {
                 </span>
               </LWTooltip>
             </div>
+            <div>
+              Sort by: <span className={classNames(classes.sortButton, {[classes.sortSelected]: contentSort === "baseScore"})} onClick={() => setContentSort("baseScore")}>
+                karma
+              </span>
+              <span className={classNames(classes.sortButton, {[classes.sortSelected]: contentSort === "postedAt"})} onClick={() => setContentSort("postedAt")}>
+                postedAt
```

Comment:
_Maybe_ this text could be a more user-friendly string like "Date" or "Date posted" rather than the property name. Since it's an admin feature I'm not 100% sure though - I trust your judgement!